### DISCUSSION TITLE
Visualize reports on pull requests in Bitbucket

### DISCUSSION THREAD

**Discussion by epikyros** [Halleck45/ast-metrics#60] [Ideas]
It would be very interesting in a code review to be able to see how a pull request affects the quality metrics of the code. The averages of a large repository won't be affected enough to be relevant, but on a file by file basis it would be a great support for the reviewer. Both in absolute values for every file changed in the pull request, but also how the pull request affects them for each file. To start with I'd love to see cyclomatic complexity and maintainablity index, but then of course all kinds of metrics could be interesting.
We're using Bitbucket pipelines for our pull requests, so for me an integration to Bitbucket would be a dream.

**Comment by Halleck45** [comment]
Hi @epikyros,

excellent idea! This idea is started in #61

**Comment by Halleck45** [comment]
Hi,

I don't use bitbucket, but now we have a dedicated github action. Do you know if same thins exist in the bitbucket ecosystem ?
